controller/service.go

```diff
@@ -12,24 +12,30 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
+//Service is a type of every method
```

Comment:
```suggestion
// Service is a type of every method
```
Add a space between the`//` and the first letter -for better readability. Do the same with all comments.

Reply:
done